ckanext/datitrentinoit/model/mapping.py

```diff
@@ -100,13 +100,12 @@ def create_base_dict(guid, metadata, config):
 
     def dateformat(d):
         return d.strftime(r"%Y-%m-%d")
-        return d.isoformat()
 
-    start_date = metadata.get_anno_inizio() or '1970'
-    if len(start_date) < 4:
-        log.warn(f"Bad annoinizio found: '{start_date}'")
-        start_date = '1970'
-    created = datetime.datetime(int(start_date), 1, 1)
+    start_date = metadata.get_anno_inizio()
+    end_date = metadata.get_anno_fine()
+
+    created = datetime.date(int(start_date), 1, 1) if start_date else ''
```

Comment:
We do want the default value for the `created` field

Reply:
Can I use 1970 as default value for created field.

Reply:
Yes, that was exactly the previous behavior that was not intended to be changed